packages/server/resources/views/emails/email_verification.edge

```diff
@@ -13,7 +13,7 @@
         <mj-text font-size="24px" color="#212121" font-family="helvetica">Please verify your email using the following code: </mj-text>
 
                 <mj-text align="left" color="#F45E43" font-family="Arial, sans-serif" font-size="13px" line-height="22px" padding-bottom="5px" padding-top="15px">
-          <p style="line-height: 40px; text-align: center; margin: 10px 0;font-size:45px;color:#212121;font-family:'Times New Roman',Helvetica,Arial,sans-serif"><b>{{token}}</b></p>
+          <p data-testid="token" style="line-height: 40px; text-align: center; margin: 10px 0;font-size:45px;color:#212121;font-family:'Times New Roman',Helvetica,Arial,sans-serif"><b>{{token}}</b></p>
```

Comment:
Thoughts: I've also been doing this, after mergin this PR I'll modify slightly few things to be able to also merge mine

Reply:
Let's merge your PR first, I will update mine after.